libcob/ChangeLog

```diff
@@ -1,4 +1,13 @@
 
+2024-09-26  Denis Hugonnard-Roce <dhugonnard@yahoo.fr>
+
+	* intrinsic.c (cob_decimal_pow) Correct c++ comment style to C
+	comment style
+
+2024-09-25  Denis Hugonnard-Roce <dhugonnard@yahoo.fr>
```

Comment:
as this is an update to a not-yet-checked in change, just edit the date for the original change and drop the update